Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
+This is how you would change the second line.
```

Comment:
Reply from CF.

---
In reply to: [59596695](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r59596695) [](ancestors = 59596695)